Investigate k8s test issue: not ok 2 Running within memory constraints

This k8s test have been failing sometimes on the CI this week. We need to investigate...
```
not ok 2 Running within memory constraints
# (from function `waitForProcess' in file ../../.ci/lib.sh, line 164,
#  in test file k8s-memory.bats, line 47)
#   `waitForProcess "$wait_time" "$sleep_time" "$pod_status_cmd"' failed
# pod "memory-test" created
# Flag --show-all has been deprecated, will be removed in an upcoming release
# Flag --show-all has been deprecated, will be removed in an upcoming release
# Flag --show-all has been deprecated, will be removed in an upcoming release
# Flag --show-all has been deprecated, will be removed in an upcoming release
# Flag --show-all has been deprecated, will be removed in an upcoming release
# Flag --show-all has been deprecated, will be removed in an upcoming release
# Flag --show-all has been deprecated, will be removed in an upcoming release
# Flag --show-all has been deprecated, will be removed in an upcoming release
# Flag --show-all has been deprecated, will be removed in an upcoming release
# Flag --show-all has been deprecated, will be removed in an upcoming release
# Flag --show-all has been deprecated, will be removed in an upcoming release
# Flag --show-all has been deprecated, will be removed in an upcoming release
# Flag --show-all has been deprecated, will be removed in an upcoming release
# Flag --show-all has been deprecated, will be removed in an upcoming release
# Flag --show-all has been deprecated, will be removed in an upcoming release
# Flag --show-all has been deprecated, will be removed in an upcoming release
# Flag --show-all has been deprecated, will be removed in an upcoming release
# Flag --show-all has been deprecated, will be removed in an upcoming release
# Flag --show-all has been deprecated, will be removed in an upcoming release
# Flag --show-all has been deprecated, will be removed in an upcoming release
# Flag --show-all has been deprecated, will be removed in an upcoming release
# Flag --show-all has been deprecated, will be removed in an upcoming release
# Flag --show-all has been deprecated, will be removed in an upcoming release
# Flag --show-all has been deprecated, will be removed in an upcoming release
Makefile:67: recipe for target 'kubernetes' failed
```

--- a/integration/kubernetes/k8s-memory.bats
+++ b/integration/kubernetes/k8s-memory.bats
@@ -29,10 +29,9 @@ setup() {
 }
 
 @test "Running within memory constraints" {
-	skip "https://github.com/kata-containers/tests/issues/680"	
 	memory_limit_size="200Mi"
-	allocated_size="150M"
-	wait_time=120
+	allocated_size="100M"
+	wait_time=300
 	sleep_time=5
 	# Create test .yaml
         sed \

